example/subelements/Makefile.am

```
@@ -0,0 +1,10 @@
+# This file is part of t8code
+# Non-recursive Makefile.am in example/basic
+# Included from toplevel directory
+
+bin_PROGRAMS += \
+	example/subelements/t8_basic_quad_refinement \
+	example/subelements/t8_test_element_functions
+
+example_subelements_t8_basic_quad_refinement_SOURCES = example/subelements/basic_quad_refinement.cxx
+example_subelements_t8_test_element_functions_SOURCES = example/subelements/test_element_functions.cxx
```

The source files should have a t8 prefix
```suggestion
example_subelements_t8_basic_quad_refinement_SOURCES = example/subelements/t8_basic_quad_refinement.cxx
example_subelements_t8_test_element_functions_SOURCES = example/subelements/t8_test_element_functions.cxx
```

---

done